op-e2e/actions/user_test.go

```diff
@@ -125,6 +133,9 @@ func runCrossLayerUserTest(gt *testing.T, test hardforkScheduledTest) {
 	if test.ecotoneTime != nil {
 		require.Zero(t, uint64(*test.ecotoneTime)%uint64(dp.DeployConfig.L2BlockTime), "ecotone fork must be aligned")
 	}
+	if test.kromaMPTTime != nil {
+		require.Zero(t, uint64(*test.kromaMPTTime)%uint64(dp.DeployConfig.L2BlockTime), "kroma mpt fork must be aligned")
+	}
```

Comment:
@Pangssu I just added this, but I wonder if kromaMPTTime should be set to exactly the timestamp of specific L2 block or can be the middle timestamp between two L2 blocks.

Reply:
It must be the exact timestamp of the L2 block.

Reply:
It's just for my curiosity, why should it be the exact timestamp?